Maple2.Database/Model/Event/GameEvent.cs

```diff
@@ -99,10 +102,24 @@ internal abstract class GameEventInfo {
                 Entries = blueMarble.Entries,
                 Tiles = blueMarble.Tiles,
             },
+            Maple2.Model.Game.Event.AttendGift attendGift => new AttendGift {
```

Comment:
I don't think we need any of the conversions in this file because it's actually duplicating the existing implicit constructors. You should be able to invoke the conversion by doing  a cast similar to:

https://github.com/kOchirasu/Maple2/blob/d98fead758a0d0eea2ef4ea6d29032f80fdb1e26/Maple2.Database/Model/Item/Item.cs#L61-L65